pkg/mapper/examples/even_odd/Makefile

```
@@ -1,10 +1,18 @@
+TAG ?= stable
+PUSH ?= false
+
 .PHONY: build
 build:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -o ./dist/even-odd-example main.go
 
 .PHONY: image
 image: build
-	docker build -t "quay.io/numaio/numaflow-go/map-even-odd:stable" --target even-odd .
+	docker buildx build -t "quay.io/numaio/numaflow-go/map-even-odd:${TAG}" --platform linux/amd64,linux/arm64 --target even-odd . --push
+
+.PHONY: test_image
```

```suggestion
.PHONY: image
```

This is image, we don't need to speculate how the image is going to be used.

Same for all other Makefiles

Update the development.md accordingly.